mysql-test/suite/rocksdb_rpl/r/optimize_myrocks_replace_info.result

```diff
@@ -0,0 +1,52 @@
+include/master-slave.inc
+Warnings:
+Note	####	Sending passwords in plain text without SSL/TLS is extremely insecure.
+Note	####	Storing MySQL user name or password information in the master info repository is not secure and is therefore not recommended. Please consider using the USER and PASSWORD connection options for START SLAVE; see the 'START SLAVE Syntax' in the MySQL Manual for more information.
+[connection master]
+create table t1(c1 int,c2 int, primary key (c1)) engine=rocksdb;
+insert into t1 values(1,1),(2,2),(3,3);
+select * from t1;
+c1	c2
+1	1
+2	2
+3	3
+replace into t1 values(1,11);
+replace into t1 values(2,22);
+replace into t1 values(3,33);
+select * from t1;
+c1	c2
+1	11
+2	22
+3	33
+replace into t1 values(2,44),(3,55);
+show binlog events;
```

Comment:
Outputs of "show binlog events" may change between versions or build types, so they make tests unstable. Instead, how about checking Get count from perf context for each replace, in both master and slave? Expected behavior is Get count does not increase in both master and slave.

```
select value into @g from information_schema.rocksdb_perf_context_global where stat_type='GET_FROM_MEMTABLE_COUNT';

REPLACE...

select case when value-@c > 0 then 'false' else 'true' end as read_free from information_schema.rocksdb_perf_context_global where stat_type='GET_FROM_MEMTABLE_COUNT';
```

Reply:
That is a good idead, 
use  -- source include/show_binlog_events.inc is also ok.

Reply:
You may combine both rocksdb_perf_context_global and source include/show_binlog_events.inc. Raw "show binlog events" caused test failures on some of our builds, since version strings were different.

```
@@ -21,7 +21,7 @@
 replace into t1 values(2,44),(3,55);
 show binlog events;
 Log_name	Pos	Event_type	Server_id	End_log_pos	Info
-master-bin.000001	4	Format_desc	1	120	Server ver: 5.6.35-log, Binlog ver: 4
+master-bin.000001	4	Format_desc	1	120	Server ver: 5.6.35-fb-log, Binlog ver: 4
```